ansible_base/lib/utils/auth.py

```diff
@@ -0,0 +1,24 @@
+from django.apps import apps as django_apps
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+
+def get_model_from_settings(setting_name):
```

Comment:
I'd encourage using type hints when writing new code:
```suggestion
def get_model_from_settings(setting_name: str) -> Any:
```